main.cpp

```diff
@@ -13,75 +13,606 @@ What you create in this project will be used as the basis of Project 5 in the co
+    
+        Manager teamManager;
+    };
+
+    It does not make sense to pass in a new Manager whenever you are going to train your team players.
+    Soccer teams have ONE manager.
+
+    a much more relevant usage would be adding a member function that hires a new manager:
+
+    struct SoccerTeam
+    {
+        struct Manager
+        {
+            ...
+        };
+
+        void hireNewManager(Manager newManager);
     
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1)
-        2)
-        3)
-    Name 2 things you can do with the:   Seat
-        1)
-        2)
+        Manager teamManager;
+    };
+
+    We can safely assume that the 'Manager' instance that is being passed in will be replacing the current 'teamManager' variable without looking at any other code.
+    This is because the function name and function argument clearly indicate what they are/what they do.
+
+    Your function names and parameter names should make LOGICAL SENSE.
+    Readers of your code should be able to INTUITIVELY understand what your function implementations will do without actually seeing the implementations.
+
+    Keep this in mind when you define your UDTs in this project part.
+    */
+};
+
+/*
+Thing 1) Market
+5 properties:
+    1) number of products in local inventory (float)
+    2) number of vehicles used in logistics (int)
+    3) number of people working in store (int)
+    4) amount of utilities (float)
+    5) daily income (float)
+3 things it can do:
+    1) Sell daily use products
+    2) Help customers with any issue
+    3) Deliver products
 */
+
+struct Market
+{
+    //number of products in local inventory
+    float numProdLocalInv = 215.3f;
+    //number of vehicles used in logistics
+    int numLogisticVehicles = 5; 
+    //number of people working in store
+    int numPeopleWorkingAtStore =32; 
+    //amount of utilities
+    float basicUtilities = 320.53f;  
+    //daily income
+    float dailyIncome = 2750; 
+
+    struct Customer
+    {
+        int age = 38;
+        bool member = true;
+        int visitsPerWeek = 7;
+        std::string gender = "Male";
+        std::string transportation = "Scooter";
+
+        void eatFood(int Frequency = 5);
+        float calculateWorkedTime(bool atHome = true, 
+                                  float time = 8.5f); 
+        float paymonthlyShould(float rent = 1500,  
+        float food = 450, 
+        float fun = 120,
+        float other = 230);
```

Comment:
`paymonthlyShould` is not an action
align the parameters with the `(`
- [ ] pick a better function name
- [ ] revise